primitives/python/qrmi_primitives/qiskit_qrmi_primitives/service.py

```diff
@@ -49,8 +50,8 @@ def __init__(self):
                 qrmi = QuantumResource(qpu, ResourceType.IBMDirectAccess)
             elif qpu_types[i] == "qiskit-runtime-service":
                 qrmi = QuantumResource(qpu, ResourceType.IBMQiskitRuntimeService)
-            # elif qpu_types[i] == "pasqal-cloud":
-            #   qrmi = QuantumResource(qpu, ResourceType.PasqalCloud)
+            elif qpu_types[i] == "pasqal-cloud":
+                qrmi = QuantumResource(qpu, ResourceType.PasqalCloud)
```

Comment:
In the end I've duplicated this file.
I'm not quite sure I know what is the best path forward, short term